.github/workflows/helm.sh

```diff
@@ -18,22 +13,22 @@ case "$subcommand" in
       helm dependency update ./helm/
       helm lint --strict ./helm/
       helm template ./helm/
-      cd $ROOT_PROJECT_DIR
+      cd ..
     done
     ;;
   package)
-    CHART_VERSION=$(git describe --abbrev=0)
+    CHART_VERSION=$(echo ${GITHUB_REF} | cut -d/ -f 3)
```

Comment:
Here `GITHUB_REF` will be tag, right?

Reply:
GITHUB_REF will be something like ` GITHUB_REF=refs/tags/0.5.0` so we are extracting tag from that.